src/base/static/utils/expression/expression.d.ts

```diff
@@ -0,0 +1,33 @@
+import PropTypes from "prop-types";
+
+import { placePropType } from "../../state/ducks/places";
+
+export interface Expression {
+  property?: string;
+  evaluate: (
+    evaluationContext: IEvaluationContext,
+  ) => number | string | boolean | Expression;
+}
+
+type Place = PropTypes.InferProps<typeof placePropType>;
```

Comment:
I think this type definition belongs in a `state/ducks/places.d.ts` file, and imported into this file if needed. 

Also, I think we should define a TS type separately instead of inferring the type from the prop type. Eventually all of our prop types will be deleted in favor of using TS, since prop types are redundant now that we're using TS.